src/plone/importexport/browser/templates/importexport.pt

```diff
@@ -81,27 +72,26 @@ i18n:domain="plone">
 
         <form action="export" method="POST">
 
-          <label> Specify Export type </label>
-          <br>
+          <label for="exportFormat"> Custom Export Settings 
+          <div class="formHelp">
+           Specify fields to export.
+           If nothing is selected, all fields will be exported
+           (@type, path, id and UID are always exported).
```

Comment:
I think its up to the user. if they don't want @type, path, and UID exported then they shouldn't have to.
Obviously the import would fail or there would documented defaults for type and path.

Also why is id seperate from path?